React/props.md

```diff
@@ -0,0 +1,59 @@
+# Passing data using `props`
+
+## Learning Goals
+- Examine how React allow components to manage data
```

Comment:
allow -> allows

Also, perhaps "use data" or "receive data" would be better than "manage data", since props doesn't allow the component to modify the data at all.